backend/storage/discussion-board.sql

```diff
@@ -0,0 +1,86 @@
+CREATE TYPE position_types AS ENUM ('Student', 'TA', 'Instructor');
+
+CREATE TABLE users (
+  user_id SERIAL PRIMARY KEY,
+  first_name varchar(50),
+  last_name varchar(50),
+  preferred_name varchar(50),
+  email varchar(320) UNIQUE NOT NULL
+);
+
+CREATE TABLE institutions (
+  id SERIAL PRIMARY KEY,
+  name varchar(100) NOT NULL,
+  location varchar(100)
```

Comment:
This is more meta, but what is `location`, is it a string representation of the coordinates? an address?

Reply:
I think I imagined it like a drop-down menu of major cities, or "other" (custom) if their city isn't listed? (though the location fields would be empty until we decide what we want to do with the locations, bc some universities span across cities.)